<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>员工信息管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/advancedgroup48.png</normaloff>:/advancedgroup48.png</iconset>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(219, 255, 249);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #ACE6E6;
font: 900 11pt &quot;Noto Sans&quot;;
color:rgb(72, 72, 72)</string>
      </property>
      <property name="headerHidden">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="toolTip">
       <string>This employee magement</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #ACE6E6;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox">
          <property name="styleSheet">
           <string notr="true">font: 11pt &quot;Noto Sans&quot;;</string>
          </property>
          <property name="text">
           <string>全选</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="deleteButton">
          <property name="styleSheet">
           <string notr="true">QPushButton#deleteButton{

   background-color:#66CCCC;
	font: 11pt &quot;Noto Sans&quot;;

}
QPushButton#deleteButton:hover{
	background-color: rgb(114, 236, 255);
}</string>
          </property>
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="searchlineEdit">
          <property name="styleSheet">
           <string notr="true">QLineEdit#lineEdit{
border:2px solid rgb(0, 190, 190);min-height:22;
border-radius:2;
}
</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="updateButton">
          <property name="styleSheet">
           <string notr="true">QPushButton#updateButton{

   background-color:#66CCCC;
	font: 11pt &quot;Noto Sans&quot;;

}
QPushButton#updateButton:hover{
	background-color: rgb(114, 236, 255);
}</string>
          </property>
          <property name="text">
           <string>修改</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="addButton">
          <property name="styleSheet">
           <string notr="true">QPushButton#addButton{

   background-color:#66CCCC;
	font: 11pt &quot;Noto Sans&quot;;

}
QPushButton#addButton:hover{
	background-color: rgb(114, 236, 255);
}</string>
          </property>
          <property name="text">
           <string>增加</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="searchButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#searchButton{

   background-color:#66CCCC;
	font: 11pt &quot;Noto Sans&quot;;

}
QPushButton#searchButton:hover{
	background-color: rgb(114, 236, 255);
}</string>
          </property>
          <property name="text">
           <string>搜索</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="6">
         <widget class="QTableWidget" name="tableWidget">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(221, 221, 221);
selection-background-color: rgb(69, 163, 168);
border:1px solid gray;
item{

font:15px  &quot;微软雅黑&quot;;
color：#677483;

}
QHeaderView{

	background-color: rgb(89, 162, 170);
}</string>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>50</number>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>50</number>
          </attribute>
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Number</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Age</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Address</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Position</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Salary</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item row="0" column="1">
           <property name="text">
            <string>123</string>
           </property>
          </item>
          <item row="0" column="2">
           <property name="text">
            <string>张三</string>
           </property>
          </item>
          <item row="0" column="3">
           <property name="text">
            <string>20</string>
           </property>
          </item>
          <item row="0" column="4">
           <property name="text">
            <string>No124</string>
           </property>
          </item>
          <item row="0" column="5">
           <property name="text">
            <string>项目经理</string>
           </property>
          </item>
          <item row="0" column="6">
           <property name="text">
            <string>12337</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3"/>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
    <item row="0" column="0" colspan="8">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="showButton">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image:url(:/refresh.png);


hover:{
background-color: 
	color: rgb(60, 255, 226);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>226</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>0</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 900 italic 27pt &quot;Noto Sans&quot;;</string>
         </property>
         <property name="text">
          <string>Employee Management System</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="minimumSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">
border-image: url(:/supergroup48.png);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Admin</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#exitButton{
background-color:#66CCCC;
font: italic 12pt &quot;Noto Sans&quot;;


}
QPushButton#exitButton:hover{
	background-color:rgb(110, 255, 245);
	color: rgb(255, 85, 0);
	font: 900 13pt &quot;Noto Sans&quot;;
}
</string>
         </property>
         <property name="text">
          <string>EXIT</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1078</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuAbout"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
